content/en/docs/Writing policies/generate.md

```diff
@@ -89,6 +89,60 @@ spec:
         name: config-template
 ```
 
+## Generating Bindings
+
+In order for Kyverno to generate a new RoleBinding or ClusterRoleBinding resource, the service account must first be bound to the same Role or ClusterRole which you're attempting to generate. If this is not done, Kubernetes blocks the request because it sees a possible privilege escalation attempt from the Kyverno service account. This is not a Kyverno function but rather how Kubernetes RBAC is designed to work.
```

Comment:
change "the service account must first be bound to the same Role or ClusterRole which you're attempting to generate"

to "the Kyverno generate controller service account must first be bound to a ClusterRole with the same or greater permissions than the role which you're attempting to generate"

Reply:
> the Kyverno generate controller service account

I only used "service account" (perhaps "the Kyverno service acount") because according to the install manifest, there is only a single service account created for all of Kyverno. There are multiple ClusterRole and ClusterRoleBinding resources created, a couple of which specifically apply to `kyverno:generatecontroller`, but all their subjects are still the one service account.

Reply:
Yes, that is correct. It may still be unclear to a user which service account we are referring to. Another option would be to say, "In order for Kyverno to generate a new RoleBinding or ClusterRoleBinding resource, its service account must first be bound to..."

Reply:
Added https://github.com/kyverno/website/pull/54/commits/ca0c2c2a9e561a23006a15c60cecebe0998825c6 which should address.